src/cljs/chronograph_web/pages/timers/views.cljs

```diff
@@ -48,19 +48,30 @@
                                   [::timers-events/stop-timer id]
                                   [::timers-events/delete-timer id]]]))}]))
 
+(defn actions [showing-create-timer-widget?]
+  [:<>
+   [antd/button
+    {:icon    icons/ArrowLeftOutlined
+     :onClick #(rf/dispatch [::timers-events/modify-selected-date -1])}]
+   [antd/button
+    {:icon    icons/ArrowRightOutlined
+     :onClick #(rf/dispatch [::timers-events/modify-selected-date 1])}]
+   [antd/date-picker {:value (time/calendar-date->moment-date @(rf/subscribe [::timers-subs/selected-date]))
+                      :onChange #(rf/dispatch [::timers-events/calendar-select-date %])}]
```

Comment:
Same as above, we can wrap this so that the callback is passed a calendar date instead of a moment.